Start the inventory section with 3 first recommendations (#8)
redhat-cop/automation-good-practices: inventories/README.adoc

This gives you a complete model of the environment to be automated, with limited effort to maintain it, and no confusion about where to modify it to get the result you need.
====

.[big]#**Differentiate clearly between "As-Is" and "To-Be" information**#
[%collapsible]
====
Explanations::
As you combine multiple sources, some will represent:
+
* discovered information grabbed from the existing environment, this is the "As-Is" information.
* managed information entered in a tool, expressing the state to be reached, hence the "To-Be" information.
+
In general, the focus of an inventory is on the managed information because it represents the desired state you want to reach with your automation. This said, some discovered information is required for the automation to work.

Rationale::
Mixing up these two kind of information can lead to your automation taking the wrong course of action by thinking that the current situation is aligned with the desired state.
That can make your automation go awry and your automation engineers confused.
There is a reason why Ansible makes the difference between "facts" (As-Is) and "variables" (To-Be), and so should you.

> Automation is at the end to make sure that the As-Is situation complies to the To-Be description.

Reviewer: How about:

```suggestion
In the end, automation is making sure that the As-Is situation complies to the To-Be description.
```

Author: Your English is generally better than mine. Done.
